examples/platform/linux/Options.cpp

```diff
@@ -793,6 +818,25 @@ bool HandleOption(const char * aProgram, OptionSet * aOptions, int aIdentifier,
         break;
     }
 #endif
+    case kDeviceOption_VendorName:
+        LinuxDeviceOptions::GetInstance().vendorName.SetValue(std::string{ aValue });
```

Comment:
How are these values then used? I cannot see this being used in `DeviceInstanceInfoProvider`.